mgmtfn/dockcontivnet/main.go

```diff
@@ -0,0 +1,474 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"os/exec"
+	"time"
+
+	"github.com/docker/docker/pkg/plugins"
+	"github.com/docker/libnetwork/drivers/remote/api"
+	"github.com/gorilla/mux"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+var (
+	optListen = flag.String("listen", "localhost:4545", "host:port to listen on")
+	optDebug  = flag.Bool("debug", false, "Get debug logging")
+)
+
+func init() {
+	flag.Usage = usage
+}
+
+func usage() {
+	fmt.Printf("Usage: " + os.Args[0] + " <interface-name> <tenant> <network>\n")
```

Comment:
I think this is still following one plugin instance per tenant per network model, which I think can change assuming https://github.com/docker/libnetwork/pull/222 PR will make it to docker, so we can start testing with labels.

This goes back to my earlier comment on us vendoring the experimental docker and start identifying/pushing for the interesting PRs. Looks like the labels PR has merge conflicts so it might need more work before we can start using it but we do need it, right?